### ISSUE TITLE
Link alias name resolution failed.

### ISSUE THREAD

**Issue by vburenin** [vmware/vic#7428]
**User Statement:**

Created link alias to a running container was not resolved from container.
```
docker run -ti busybox ping cont7alias 
ping: bad address 'cont7alias'
```

**Details:**
I was not able to reproduce it on my VC setup. Everything worked as expected.

![image](https://user-images.githubusercontent.com/4350891/36816710-3f02fd8e-1ca4-11e8-8e94-71a05985e25e.png)

[Test-Cases.Group1-Docker-Commands.1-36-Docker-Rename-VCH-17099-4617-container-logs.zip](https://github.com/vmware/vic/files/1769082/Test-Cases.Group1-Docker-Commands.1-36-Docker-Rename-VCH-17099-4617-container-logs.zip)

**Comment by mdubya66** [comment]
what happens if you block 8.8.8.8

**Comment by vburenin** [comment]
This issue is not related to Google DNS. It is potentially due to the same two VCHs on the same network with the same IP address. 
@mhagen-vmware  I heard you addressed that issue just yesterday. Right?

**Comment by mhagen-vmware** [comment]
Still in process: https://github.com/vmware/vic/pull/7413

**Comment by mhagen-vmware** [comment]
not same IP address, but sharing bridge network

**Comment by mhagen-vmware** [comment]
but I am also watching these builds and the point at this which this test failed would be very unlikely to have 2 VCH installed.

**Comment by mhagen-vmware** [comment]
This build should have the unique bridges working:
https://ci-vic.vmware.com/vmware/vic/17145/8

**Comment by mhagen-vmware** [comment]
This is apparently fixed with #7413

**Comment by vburenin** [comment]
@mhagen-vmware Is it possible that containers can't see VCH for some reason? I didn't see any DNS request related to those alias link names.

I tested it on my VC setup 5 times by now and everything works just fine.
However, I tested it against manually deployed VCH.
